dashboard/app/controllers/levels_controller.rb

```diff
@@ -203,6 +204,10 @@ def embed_level
     render 'levels/show'
   end
 
+  def set_applab_user_id
+    @applab_user_id = applab_user_id
```

Comment:
Because it doesn't work if try to set @applab_user_id in LevelsHelper :) Note that I'm doing this similarly to how other instance @variables are set in controllers, e.g. @callback is accessed in LevelsHelper but is defined here: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/level_sources_controller.rb#L10

@laurelfan , can you explain why this pattern is used / needed for initializing variables used in LevelsHelper?